sdk/search/Azure.Search/src/Utilities/Constants.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Search
+{
+    /// <summary>
+    /// Constants used by the Search client library.
+    /// </summary>
+    internal static class Constants
+    {
+        // TODO: XXXXX - Switch constants to use JsonEncodedText
```

Comment:
Tip: You might keep the consts for equality, but then add `JsonEncodedText` static readonlys for writing.

Reply:
I believe we can use the encoded text for equality as well, but I'll be following up separately on all of that.